src/engine/renderer/tr_image.cpp

```diff
@@ -2771,21 +2771,6 @@ void R_CreateBuiltinImages()
 
 	tr.blackImage = R_CreateImage( "$black", ( const byte ** ) &dataPtr, 1, 1, 1, imageParams );
 
-	// red
-	Vector4Set( data, 255, 0, 0, 255 );
-
-	tr.redImage = R_CreateImage( "$red", ( const byte ** ) &dataPtr, 1, 1, 1, imageParams );
```

Comment:
Also delete these fields from the `tr` struct

Reply:
Done.